source/wp-content/themes/wporg-developer/inc/template-tags.php

```diff
@@ -707,7 +707,8 @@ function get_signature( $post_id = null ) {
 		$types        = array();
 
 		if ( 'wp-parser-class' === get_post_type( $post_id ) ) {
-			return $signature;
+			/* translators: %s: The name of the class. */
+			return sprintf( __( 'class %s {}', 'wporg'), $signature );
```

Comment:
I know I just approved this, but should this be translatable? `class Blah {}` should always be English because it's PHP, right?

Reply:
In https://github.com/WordPress/wporg-developer/pull/72/commits/d4d1b8cdd4da3d050af79d0f3b4a84cffdff4513, I removed the translation and also added a span wrapper around `class`, so we can style it like the `apply_filters`/etc — so it's visible, but a little less visually important.

<img width="582" alt="Screen Shot 2022-06-03 at 12 14 57 PM" src="https://user-images.githubusercontent.com/541093/171904825-5e0eed63-6d11-4232-96f3-a206933afb91.png">